src/sagemaker/image_uri_config/huggingface.json

```diff
@@ -4,7 +4,8 @@
         "version_aliases": {
             "4.4": "4.4.2",
             "4.5": "4.5.0",
-            "4.6": "4.6.1"
+            "4.6": "4.6.1",
+            "4.10": "4.10.2"
```

Comment:
nit: Possibly for next time, it would be good to add in comma here to not update unrelated lines(line 7) in code review.

`"4.10": "4.10.2",`

Reply:
added